client.go

```diff
@@ -503,7 +503,7 @@ func (client *Client) processEvent(event *Event, hint *EventHint, scope EventMod
 		options.SampleRate = 1.0
 	}
 
-	if !sample(options.SampleRate) {
+	if event.Type != transactionType && !sample(options.SampleRate) {
```

Comment:
Can we leave a comment here that transactions use their own sampler/samplerate?

Reply:
Done.